src/org/opensolaris/opengrok/search/Results.java

```diff
@@ -263,6 +254,46 @@ public static void prettyPrint(Writer out, SearchHelper sh, int start,
         }
     }
 
+    private static void printPlain(SearchHelper searcHelper,
+        RuntimeEnvironment env, Document doc, int docId, Writer out,
```

Comment:
Could you please consider somehow combining the method parameters? This violates current checkstyle parameter number setting which is set to default value, right now it is `7`.

Reply:
ok revised to use a private container object for the arguments to `printPlain()` that do not change in the calling loop.